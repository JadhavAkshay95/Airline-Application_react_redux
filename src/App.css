.App {
  text-align: center;
}

.App-header {
  display: flex;
  flex-direction: column;
  align-items: center;
  justify-content: center;
  font-size: calc(10px + 2vmin);
  color: white;
}

.header {
  background-color: #00b383;
  background: linear-gradient(to right, #1affc2 0%, #00b383 50%);
  height: 70px;
  text-align: center;
  padding: 10px 0px;
}
