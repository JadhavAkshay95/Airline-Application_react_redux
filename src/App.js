import React from 'react';
import './App.css';
import Footer from './component/common/Footer';
import Routes from './Routes';

function App() {
  return (
    <div className="App">
      <Routes />
      <Footer />
    </div>
  );
}

export default App;
