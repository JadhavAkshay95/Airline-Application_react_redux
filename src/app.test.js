import React from 'react'
import { shallow } from 'enzyme'
import App from './App'

const renderAppComponent = args => {
  const defaultProps = {}
  const props = { ...defaultProps, ...args }
  return shallow(<App {...props} />)
}

describe('Render app component', () => {
  it('Render login modal', () => {
    const wrapper = renderAppComponent()
    expect(wrapper).toMatchSnapshot()
  })
})
