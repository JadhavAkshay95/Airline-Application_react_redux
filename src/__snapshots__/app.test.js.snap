// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render app component Render login modal 1`] = `ShallowWrapper {}`;
